### DISCUSSION TITLE
How do i configure to use fpu?

### DISCUSSION THREAD

**Discussion by C64HexCoder** [prb28/vscode-amiga-assembly#235] [General]
I'm working on a demo and i wanna use fpu. but when i run fpu codes the demo crashes. How do i configure the script so the amiga 1200 have fpu or change the cpu it uses to 68030 or heigher?

**Comment by prb28** [comment]
You have to change the emulator configuration and the vasm build options.
Do you use winuae or fa-uae ?

**Comment by C64HexCoder** [comment]
Thanks for the reslond. I've succeeded doing it with both both winuae and
fs-uae. For the first one i changed the settings (runing it and pressing
f12) but for the second one i couldn't finf a way to chane the settins. So
i copied it drom my fs-uae, changed its name to default.fs-uae and it did
not work. Not even when telling the fs-uae to use it on the command line.
So i guessed the only way to do so is by changing the compiler script. So i
guessed and added "--fpu=68882", and it worked. Shame that there is no
documentation about it.

בתאריך יום ה׳, 18 באוג׳ 2022, 08:21, מאת prb28 ***@***.***>:
